### PR TITLE
Update HTTPS and Storage Explorer instructions.

### PR DESCRIPTION
**Pull request by jongio**
Thanks for contribution! Please go through following checklist before sending PR.

### PR Branch Destination

- For Azurite V3, please send PR to `dev` branch.
- For legacy Azurite V2, please send PR to `legacy-dev` branch.

### Always Add Test Cases

Make sure test cases are added to cover the code change.

### Development Guideline

Please go to CONTRIBUTION.md for steps about setting up development environment and recommended Visual Studio Code extensions.

### CODE REVIEW

**Comment by jongio** [comment]
I will update this PR in a moment. Please don't review yet.

**Comment by jongio** [comment]
@XiaoningLiu - This is ready to be reviewed.  Thank you.

File: README.md [hunk 1]
```diff
@@ -445,43 +447,90 @@ Or if the SDK or tools support following short connection string:
...
 
-The full https connection string is:
+The full HTTPS connection string is:
 
 ```
 DefaultEndpointsProtocol=https;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=https://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=https://127.0.0.1:10001/devstoreaccount1;
 ```
 
-Take blob service only, the https connection string is:
+To use the Blob service only, the HTTP connection string is:
```
Comment (XiaoningLiu) [inline README.md:458]:
```suggestion
To use the Blob service only, the HTTPS connection string is:
```

File: README.md [hunk 2]
```diff
@@ -445,43 +447,90 @@ Or if the SDK or tools support following short connection string:
...
 
-#### Storage Explorer with HTTPS
+#### Storage Explorer with Azurite HTTPS
+
+By default Storage Explorer will not open an HTTPS endpoint that uses a self-signed certificate. If you are running Azurite with HTTPS, then you are likely using a self-signed certificate. Fortunately, Storage Explorer allows you to import SSL certificates via the Edit -> SSL Certificates -> Import Certificates dialog.
 
-NOTE: Storage Explorer is a Node.js application that does not work with a local CA, so if you are using a local CA, then you need to set the following environment variable.
+##### Import Root CA
+
+1. Find the "Root Certificate aka Root CA" on your local machine.
```
Comment (XiaoningLiu) [inline README.md:512]: Root CA by mkcert tool? Where can we usually find it?

**Comment by XiaoningLiu** [comment]
Will merge to dev and sync to master

**Pull request merged**